power-management/common/libpwmgmt.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Red Hat, Inc. All rights reserved.
+#
+# This copyrighted material is made available to anyone wishing
+# to use, modify, copy, or redistribute it subject to the terms
+# and conditions of the GNU General Public License version 2.
+#
+# This program is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public
+# License along with this program; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+# Boston, MA 02110-1301, USA.
+#
+
+source $(dirname $(readlink -f $BASH_SOURCE))/../../cki_lib/libcki.sh
+
+#
+# Check CPU vendor is Intel or not
+#
+function is_intel
+{
+    typeset vendor=$(grep vendor /proc/cpuinfo | sort -u | awk '{print $3}')
```

Comment:
Is the `sort -u` _only_ used to get the unique lines? If so, you can also use `uniq`, which is clearer in this context I'd say (because there's nothing to sort if they're all the same).

Reply:
OK, use `uniq` instead of `sort -u`.